Dockerfile

```diff
@@ -81,8 +82,20 @@ ARG REACT_APP_GTM_PREVIEW
 # Build application
 RUN yarn build
 
+# ==========================================
+FROM helsinkitest/node:12-slim as production
+# ==========================================
+# Use non-root user
+USER appuser
+
+COPY --chown=appuser:appuser package.prod.json /app/package.json
```

Comment:
You could also copy yarn.lock or use specific canvas version to always use right canvas version

Reply:
I thought I removed "^" from the package.prod.json. But it seems to be there. Do you think that is enough to have the correct version? Would copying yarn.lock only install canvas or all the dependencies?

Reply:
Removing ^ should be enough. Good question about yarn.lock. I think no, but to be sure I have to test it.

Reply:
Copying yarn.lock wan't good idea :D 
<img width="1163" alt="Screenshot 2020-12-19 at 14 33 54" src="https://user-images.githubusercontent.com/24706814/102689503-4719a780-4207-11eb-9aa4-4b103d845d7e.png">